sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/InsertIntoHadoopFsRelationCommand.scala

```diff
@@ -227,10 +227,8 @@ case class InsertIntoHadoopFsRelationCommand(
     val staticPartitionPrefix = if (staticPartitions.nonEmpty) {
       "/" + partitionColumns.flatMap { p =>
         staticPartitions.get(p.name) match {
-          case Some(value) =>
-            Some(escapePathName(p.name) + "=" + escapePathName(value))
-          case None =>
-            None
+          case Some(value) => Some(getPartitionPathString(p.name, value))
```

Comment:
Please get rid of this dangling `None` with the following:

```
staticPartitions.get(p.name).map(value => getPartitionPathString(p.name, value))
```

Reply:
thanks and the `value` can be reduced too